src/app/courses-page/pipes/filter.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Course } from '@app/models';
+
+@Pipe({
+  name: 'filter'
+})
+export class FilterPipe implements PipeTransform {
+  transform(arr: Course[], condition: string): Course[] {
+    return arr.filter(course => course.title.toLowerCase().includes(condition.toLowerCase()));
```

Comment:
nit:
to improve performance you can move `condition.toLowerCase()` out of the filter loop to transform it to lower case just once before iteration

Reply:
@SteveRusin Changed.
Thanks)